webrtc-stats.html

```diff
@@ -1218,6 +1218,16 @@
              double                        roundTripTime;
              double                        availableOutgoingBitrate;
              double                        availableIncomingBitrate;
+             unsigned long long            requestsReceived;
+             unsigned long long            requestsSent;
+             unsigned long long            responsesReceived;
+             unsigned long long            responsesSent;
+             unsigned long long            retransmissionsSent;
```

Comment:
If an implementation never uses retransmissions (and just schedules another ICE check at the next time one should be scheduled), then will this always be zero?  That would be the case with our ICE implementation.